frontend/src/concepts/pipelines/content/configurePipelinesServer/PipelineDropdown.tsx

```diff
@@ -135,9 +136,9 @@ export const PipelineDropdown = ({
                       '•••••••••••••••••'
                     )
                   }
-                  itemId={dataItem.data.metadata.name}
+                  itemId={dataItem.metadata.name}
                 >
-                  {getDataConnectionDisplayName(dataItem)}
+                  {getConnectionTypeDisplayName(dataItem)}
```

Comment:
```suggestion
                  {getDisplayNameFromK8sResource(dataItem)}
```

Similarly, since it's a connection (I think) get the display name of the connection, not the type. 

This code as is would probably display "s3"